packages/sdk/python/src/relayfile/self_host_connect.py

```
@@ -0,0 +1,144 @@
+        end_user_id: str,
+        connection_id: str | None = None,
+        metadata: dict[str, Any] | None = None,
+    ) -> SelfHostConnectResult:
+        normalized = _normalize_relayfile_provider(relayfile_provider)
+        provider_config_key = self._resolve_provider_config_key(normalized)
+        connect_provider = self._require_connect_provider()
+        trimmed_end_user_id = end_user_id.strip()
+        if not trimmed_end_user_id:
+            raise ValueError("end_user_id is required to start a self-host connect session")
+
+        input: CreateConnectSessionInput = {
```

Ruff (0.15.18)

[error] Variable `input` is shadowing a Python builtin

(A001)